doc/method/results.md

```diff
@@ -0,0 +1,342 @@
+
+# Results
+
+## Alignment
+
+The alignment procedure examines every source node.  The following is
+a breakdown of the disposition of source nodes, across all sources, as
```

Comment:
We aren't trying to 'dispose' of the nodes. We are trying to match / align them.